src/site/markdown/download.md.vm

```diff
@@ -0,0 +1,84 @@
+---
+title: Download ${project.name} Source
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Download ${project.name} ${project.version} Source
+
+${project.name} ${project.version} is distributed in source format\. Use a source archive if you intend to build
+${project.name} yourself\. Otherwise, simply use the ready\-made binary artifacts from central repository\.
+
+You will be prompted for a mirror \- if the file is not found on yours, please be patient, as it may take 24
+hours to reach all mirrors\.
+
+In order to guard against corrupted downloads/installations, it is highly recommended to
+[verify the signature](http://www.apache.org/dev/release-signing#verifying-signature)
+of the release bundles against the public [KEYS](https://www.apache.org/dist/maven/KEYS) used by the Apache Maven
+developers\.
+
+${project.name} is distributed under the [Apache License, version 2\.0](http://www.apache.org/licenses/)\.
+
+We **strongly** encourage our users to configure a Maven repository mirror closer to their location, please read [How to Use Mirrors for Repositories](./guides/mini/guide-mirror-settings.html)\.
+
+<a id="mirror"></a>
+
+${esc.h}${esc.h} Mirror
+
+\[if-any logo\]
```

Comment:
@slawekjaranowski Is this still supposed to be used like this? I am not sure which placeholders are replaced by the CGI (documentation at https://infra.apache.org/release-download-pages.html#custom is very sparse). Mirrors are gone nowadays...

Reply:
There is just very few placeholders available for use in the template: https://maven.apache.org/skins/maven-fluido-skin/download.cgi?asjson=true, the templating language is described in https://github.com/gstein/ezt/blob/wiki/Syntax.md.

Reply:
First I would like to update download page like in https://github.com/apache/maven-changes-plugin/pull/68
we will not a special cgi for it

Reply:
https://issues.apache.org/jira/browse/MSKINS-240

Reply:
As a globally provided template is some more efforted I refactored now the download page to a more simple version (no longer mentioning any mirrors). However it still relies on the placeholder mechanism of the underlying [closer.lua script](https://github.com/apache/infrastructure-p6/blob/production/modules/closer_cgi/files/closer.lua) due to reasons outlined at https://issues.apache.org/jira/browse/MSKINS-240?focusedCommentId=17902720&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-17902720